style.css

```diff
@@ -65,3 +67,150 @@ body {
   width: 100%;
   height: auto;
 }
+
+.section-2 {
+  margin-top: 0;
+  margin-bottom: -6px;
+}
+
+.projects-card {
+  list-style: none;
+  display: flex;
+  justify-content: start;
+  margin-left: 16px;
+}
+
+.projects-heading-2 {
+  margin-left: 16px;
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  line-height: 16px;
+  font-size: 20px;
+  padding: 26px 0;
+}
+
+.projects-img {
+  width: 100%;
+  height: auto;
+  margin-top: -2px;
+}
+
+.project-button {
+  margin-left: 16px;
+  margin-top: 15px;
+  margin-right: 30px;
+  padding: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  color: white;
+  background-color: #1a2236;
+  border: 2px solid white;
+}
+
+.project-button:hover {
+  cursor: pointer;
+}
+
+.project-button::after {
+  cursor: pointer;
+}
+
+.list-btn {
+  color: #1a2236;
+  background-color: #98dede;
+  font-size: 10.33px;
+  margin-right: 10px;
+  border: none;
+  padding: 5px;
+  margin-top: 12px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.background-color {
```

Comment:
- [ ] Nice work to use grid as required. Kindly, change this class name with a descriptive name. Imagine calling `background-color class` with background-color property.